web/app/js/components/GrafanaLink.jsx

```diff
@@ -3,30 +3,62 @@ import React from 'react';
 import _isEmpty from 'lodash/isEmpty';
 import { grafanaIcon } from './util/SvgWrappers.jsx';
 
-const GrafanaLink = function({ PrefixedLink, name, namespace, resource }) {
+const GrafanaLink = function({ PrefixedLink, name, namespace, resource, grafanaExternalUrl, grafanaPrefix }) {
   let link = `/grafana/d/linkerd-${resource}?var-${resource}=${name}`;
```

Comment:
nit: How about we attach `linkerd-${resource}?var-${resource}=${name}` after the if condition below? so as to not repeat this twice i.e inside the if and before.

Reply:
@Pothulapati good idea, I have pushed a change to implement your suggestion.